static/app/components/superuserStaffAccessForm.tsx

```diff
@@ -163,9 +163,9 @@ class SuperuserStaffAccessForm extends Component<Props, State> {
   };
 
   handleLogout = () => {
-    const {superuserUrl} = window.__initialData?.links;
+    const {superuserUrl} = window.__initialData.links;
```

Comment:
`window.__initialData` is typed as being Non-Nullable here:

https://github.com/getsentry/sentry/blob/4e59804c08f3d27d705c33f1e3c5d630bd0a57c1/static/app/types/system.tsx#L51

I was thinking of updating the type, but more places assume its non nullable. For this instance, it's inside a staff superuser form, so even if I get it wrong here the impact should be low